config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```diff
@@ -1270,3 +1270,55 @@ presubmits:
           requests:
             cpu: 4
             memory: 6Gi
+
+  # This jobs runs e2e.test with a focus on tests for the Dynamic Resource Allocation feature (currently alpha).
+  # crio >= 1.23.0 has the necessary support for Container Device Interface (CDI).
+  - name: pull-kubernetes-node-crio-dra
+    cluster: k8s-infra-prow-build
+    skip_branches:
+    - release-\d+\.\d+  # per-release image
+    annotations:
+      testgrid-dashboards: sig-node-presubmits
+      testgrid-tab-name: pr-crio-dra
+    # Not relevant for most PRs.
+    always_run: false
+    # This covers most of the code related to dynamic resource allocation.
+    run_if_changed: /dra/|/dynamicresources/|/resourceclaim/|/resourceclass/|/podscheduling/
+    # If it runs, it has to pass.
+    optional: false
```

Comment:
This is debatable. It should be safe because typically the job won't run, but I can also see that this might be too early for tests of an alpha feature.